cishouseholds/derive.py

```diff
@@ -33,14 +31,17 @@ def assign_date_from_filename(df: DataFrame, column_name_to_assign: str, filenam
     return df
 
 
-def assign_visit_order_from_digital(df: DataFrame, column_name_to_assign: str, visit_date_column: str, id_column: str):
+def assign_visit_order(df: DataFrame, column_name_to_assign: str, visit_date_column: str, id_column: str):
     """
     assign an incremental count to each participants visit
+    column_name_to_assign: column to show count
```

Comment:
This doesn't follow the [numpy-doc docstring format](https://numpydoc.readthedocs.io/en/latest/format.html) standard that we use. Please see other functions for an example on how to add the `Parameters` section of the docstring